### PR TITLE
fix(main/elixir): Match source tarball with current erlang version

### LINKED ISSUE #21204: Elixir mix can not install elixir packages
I have elixir and erlang installed:

```
Erlang/OTP 27 [erts-15.0.1] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [jit]

Elixir 1.17.2 (compiled with Erlang/OTP 25)
```

When I tried to install an elixir:

```sh 
mix archive.install hex phx_new
```

I received the following error:

```
** (EXIT from #PID<0.95.0>) an exception was raised:
    ** (CaseClauseError) no case clause matching: :undefined
        (public_key 1.16.1) pubkey_os_cacerts.erl:40: :pubkey_os_cacerts.get/0
        (mix 1.17.2) lib/mix/utils.ex:664: Mix.Utils.read_httpc/1
        (mix 1.17.2) lib/mix/utils.ex:576: anonymous fn/2 in Mix.Utils.read_path/2
        (elixir 1.17.2) lib/task/supervised.ex:101: Task.Supervised.invoke_mfa/2
        (elixir 1.17.2) lib/task/supervised.ex:36: Task.Supervised.reply/4


15:29:42.543 [error] Task #PID<0.111.0> started from #PID<0.95.0> terminating
** (CaseClauseError) no case clause matching: :undefined
    (public_key 1.16.1) pubkey_os_cacerts.erl:40: :pubkey_os_cacerts.get/0
    (mix 1.17.2) lib/mix/utils.ex:664: Mix.Utils.read_httpc/1
    (mix 1.17.2) lib/mix/utils.ex:576: anonymous fn/2 in Mix.Utils.read_path/2
    (elixir 1.17.2) lib/task/supervised.ex:101: Task.Supervised.invoke_mfa/2
    (elixir 1.17.2) lib/task/supervised.ex:36: Task.Supervised.reply/4
Function: #Function<5.19587555/0 in Mix.Utils.read_path/2>
    Args: []

```

### PR DESCRIPTION
**Pull request by Biswa96**
Fixes #21204

### CODE REVIEW

**Comment by Biswa96** [comment]
I would greatly appreciate reviews from shell wizards 🪄

**Comment by TomJo2000** [comment]
You called?

File: packages/elixir/build.sh [hunk 1]
```diff
@@ -3,8 +3,10 @@ TERMUX_PKG_DESCRIPTION="Elixir is a dynamic, functional language designed for bu
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="1.17.2"
-TERMUX_PKG_SRCURL=https://github.com/elixir-lang/elixir/releases/download/v${TERMUX_PKG_VERSION}/elixir-otp-25.zip
-TERMUX_PKG_SHA256=6ca8215fd8b1fa4472fa62c8258d3c2880145070bf04848f51861c7d1d219c7e
+TERMUX_PKG_REVISION=1
+_ERLANG_MAJOR_VERSION=$(. $TERMUX_SCRIPTDIR/packages/erlang/build.sh; echo ${TERMUX_PKG_VERSION%%.*})
```
Comment (TomJo2000) [inline packages/elixir/build.sh:7]: Quote yo vars.
```suggestion
_ERLANG_MAJOR_VERSION=$(. "$TERMUX_SCRIPTDIR"/packages/erlang/build.sh; echo "${TERMUX_PKG_VERSION%%.*}")
```
But that'll do it.

**Review by TomJo2000 — APPROVED** [review]

File: packages/elixir/build.sh (hunk 1 above)
Comment (Biswa96) [reply packages/elixir/build.sh]: Done.

Comment (TomJo2000) [reply packages/elixir/build.sh]: The command substitution itself should also be quoted.
Which, I recognize is overkill.
But we may as well do our quoting properly.

Comment (Imissmiya962019) [reply packages/elixir/build.sh]: > Quote yo vars.
> 
> But that'll do it.

**Review by Imissmiya962019 — COMMENTED** [review]
I'm need help please in english damit wanna learn this shit but nobody to ask questions but useless google

**Pull request merged**